dockerfiles/drbd-driver-loader/Makefile

```diff
@@ -14,7 +14,7 @@ all: update upload
 update:
 	for f in $(DF); do \
 		pd=$(PROJECT)-$$(echo $$f | sed 's/^Dockerfile\.//'); \
-		docker build --no-cache=$(NOCACHE) -f $$f -t $$pd:$(TAG) . ; \
+		docker build --build-arg DRBD_VERSION=$(shell cat DRBD_VERSION) --no-cache=$(NOCACHE) -f $$f -t $$pd:$(TAG) . ; \
```

Comment:
hm, these `cat`s are pretty fragile, usually I'm not a big fan of them. In my experience having proper variables in a file like `DRBD_VERSION='1.2.3'` and sourcing this file in the shell like `source env; echo "$DRBD_VERSION"` is ways more robust.
`x=$(cat <(echo -e "\n\nbroken")); echo "1${x}2"`